src/styles/Form.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=Montserrat&family=Roboto&family=Roboto+Slab&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.bookForm {
+  margin: 1.5rem auto;
+  border-top: 2px solid #d8bfbf;
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.addBtn {
+  height: 2.813rem;
+  width: 11.5rem;
+  margin: 19px 34px 0 0;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  border: none;
+  background-color: var(--azure);
+  font-family: 'Montserrat', serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+}
+
+.author-input {
+  width: 25%;
+  padding-left: 0.25rem;
+  font-size: 1rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  font-family: 'Montserrat', sans-serif;
+  background-color: #fff;
+}
+
+.category-input {
+  width: 15%;
+  padding-left: 0.25rem;
+  font-size: 1rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  font-family: 'Montserrat', sans-serif;
+  background-color: #fff;
+}
```

Comment:
Great job @Chimoneg27 🤝

- [ ] As stated in the project requirements, kindly ensure you do not repeat the same rule. kindly create a single class for the repeating rules and re-use this class wherever needed. You may do this for the following rules. `font-family: 'Montserrat', sans-serif;`, and `border: solid 1px #e8e8e8;`.